### PR TITLE
Remove comments and refactor

### PR DESCRIPTION
**Pull request by robertjosephderouin**
## **_PR TEMPLATE_**
1. Is this a fix or a feature?
- [x] Fix
2. Describe the feature / fix / refactor?
- [x] Removed comments
3. Describe the feature / fix / refactor?
- [x] Fixed bug where refactored variable names no longer matched.
4. Where should the reviewer start / test?
- [x] main.js

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -207,21 +207,21 @@ saveViewMainButton.addEventListener('click', renderMainPage);
...
 
 savedPosterGrid.addEventListener('dblclick', deleteSavedPoster);
 
 // functions and event handlers go here 👇
 
 function createPoster() {
-  var imageRandomNumber = getRandomIndex(images); //NEW
+  var imageRandomNumber = getRandomIndex(images);
   currentPoster = new Poster(images[imageRandomNumber].url, images[imageRandomNumber].altText, titles[getRandomIndex(titles)], quotes[getRandomIndex(quotes)]);
   renderCoverImage(currentPoster);
```
Comment (PaigeVannelli) [inline src/main.js:219]: It looks like you've added an argument here of currentPoster but don't have a parameter set in your renderCoverImage function

File: src/main.js [hunk 2]
```diff
@@ -207,21 +207,21 @@ saveViewMainButton.addEventListener('click', renderMainPage);
...
 }
 
 function renderCoverImage() {
   randomImage.src = currentPoster.imageURL;
-  randomImage.alt = currentPoster.altTxt; //NEW
+  randomImage.alt = currentPoster.altTxt;
```
Comment (PaigeVannelli) [inline src/main.js:224]: I like how you dynamically change the alt text!

File: src/main.js [hunk 3]
```diff
@@ -259,16 +259,17 @@ function savePosters() { //NEW
 function renderSavedPoster() {
   savedPosterGrid.innerHTML = "";
```
Comment (PaigeVannelli) [inline src/main.js:260]: Nice work resetting the html every time

**Review by PaigeVannelli — COMMENTED** [review]
Looks great! Really clean code and good use of SRP. Looks like there's only one small edit you might make.

File: src/main.js (hunk 1 above)
Comment (stevecalla) [reply src/main.js:219]: @robertjosephderouin We can delete the parameter in the renderCoverImage function for currentPoster. This was the idea that we might eliminate the global variables but we decided not to do that based on instructor feedback. If you can make that change, I'll merge it, and then I'll send a note to Hannah.

**Pull request merged**